lib/tiled.dart

```diff
@@ -88,7 +89,7 @@ class Tiled {
 
   Future _load() async {
     map = await _loadMap();
-    image = await Flame.images.load(map.tilesets[0].image.source);
+    image = await Flame.images.load(map.tilesets[0].image!.source!);
```

Comment:
for new tiled: `image = await Flame.images.load(map.tilesets[0].image!.source);`